cinema/views.py

```diff
@@ -136,3 +157,8 @@ def get_serializer_class(self):
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+    def get_permissions(self):
+        if self.action not in ("list", "create"):
+            raise NotFound
```

Comment:
It is not true. It is mean you can not find object by id. Which is not true in this case
<img width="796" alt="image" src="https://github.com/mate-academy/py-token-authentication/assets/65166203/ba32ad83-2178-4653-9f62-dda686c2370b">